tests/e2e_tests/client_utils.py

```
@@ -0,0 +1,333 @@
+import hashlib
+import io
+import random
+import requests
+import time
+import uuid
+from typing import Optional, Dict, Any, List, Generator, Tuple
+
+from qdrant_client import QdrantClient, models
+from qdrant_client.http.exceptions import UnexpectedResponse
```

Ruff (0.11.9)

`qdrant_client.http.exceptions.UnexpectedResponse` imported but unused

Remove unused import: `qdrant_client.http.exceptions.UnexpectedResponse`

(F401)